R/translate.R

```diff
@@ -60,10 +60,19 @@ deepl_translate <- function(path,
     formality,
     values = c("default", "more", "less", "prefer_more", "prefer_less")
   )
-
-  # protect Hugo shortcodes ----
   temp_markdown_file <- withr::local_tempfile()
   markdown_lines <- brio::read_lines(path)
+  temp_markdown_file <- withr::local_tempfile()
+
+  # change on char equation to something longer and
+  # _ (used in latex) to something that is not going to be <emphed>
+
+  markdown_lines <- gsub("\\$(.)\\$", "\\$xXx\\1\\$", markdown_lines)
+  # _ is a special character in markdown and in latex, we neutralize it
+  markdown_lines <- gsub("_", "°°", markdown_lines)
```

Comment:
I prefer to handle such cases with XML as DeepL API can make some characters disappear (as used to happen with the ":" of footnotes :sweat_smile:)